docs/docs/conceptual/choosing-a-cms.md

```diff
@@ -0,0 +1,69 @@
+
+Users choosing other CMSs typically have a specific reason for their choice. While every user has different reasons for their choice, some trends we see users choosing **Sanity** or **Strapi** for the developer-friendliness or if they need something on-premise; **DatoCMS** if they will run into Contentful's model limits and want a lower-priced alternative, **Prismic** if they like the content editing UI; **Contentstack** if they like the editing UI and have an enterprise budget; **Wordpress** when the client or content team is already familiar with the Wordpress UI. **Drupal** if open-source, configurability / custom code are important.
+
+## Using multiple CMS systems together
+
+Another approach is, rather than using one CMS, use different CMSs for different parts of the website. Known as a ["content mesh" approach](https://www.gatsbyjs.com/blog/2018-10-04-journey-to-the-content-mesh/), the typical rationale here is that one CMS provides must-have specialized functionality but is less good for the rest of the site. Typically the decision is to use a specialized CMS for part of the website, and a general-purpose CMS for most of the website.
+
+Typically there are two reasons to intentionally take this approach:
+
+### Using Shopify as an e-commerce system
+
+Shopify has best-in-class e-commerce functionality, but the rest of their interface (for example, their blogs feature) is often seen as subpar compared to other systems. As a result, it's very standard to use Shopify as the backend for the store, but a general purpose CMS like Contentful as the backend for the rest of the site.
+
+### Using Wordpress as a blogging tool
+
+Because Wordpress is so familiar to content authors, and has a best-in-class content composition experience, it's quite common to move the blog portion of the website with Gatsby, and use a flexible content modelling CMS for the rest of the website.
+
+Apollo does this [for their blog](https://twitter.com/apollographql/status/1250479066605662210), as we do for the blog at [gatsbyjs.com/blog](gatsbyjs.com/blog).
+
+### Considerations when using multiple CMSs
+
+One of the key considerations when using content in multiple systems is that at some point one  content often needs to "know about" another system. For example, a landing page with content in Contentful may need to embed information about a specific product SKU (which lives in Shopify).
+
+The easiest way to create relationship references across CMSs is through one CMS storing unique IDs of content living in another CMS. In this case, you'd store an array of Shopify product IDs as a field of the relevant model in Contentful, then pull in the correct data via the appropriate queries in gatsby-node.js.
```

Comment:
It would be great if we could find or build an example of a site that does this, so that if folks don't understand this explanation they could look through the code themselves.

Reply:
Or even just attach a screenshot of what that would look like in Contentful/Shopify

Reply:
That's a really good idea. I'll do this for our Use Cases pages.